BSPConversionLib/Source/EntityConverter.cs

```diff
@@ -340,7 +401,7 @@ private void GiveWeaponOnStartTouch(Entity trigger, Entity weaponEnt)
 				target = "!activator",
 				action = "GiveDFWeapon",
 				param = weaponIndex.ToString(),
-				delay = 0f,
+				delay = 0.01f, //hack to make sure that the weapon removal applies before weapon give
```

Comment:
You can leave this hack for now, but we should definitely find a better way of handling triggers that strip and give weapons simultaneously. 

Perhaps we just need to avoid calling RemoveWeaponOnStartTouch for any weapons referenced by the target_give?

Reply:
It's done this way in q3 because it resets ammo when the weapon is stripped away otherwise you can hit the trigger multiple times to add infinite ammo. I'm not sure how our ammo system works to be honest, so maybe that's not an issue in mmod.